staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -203,6 +203,13 @@ func (s *SecureServingOptions) ApplyTo(config **server.SecureServingInfo) error
 
 	if s.Listener == nil {
 		var err error
+		// BindAddress takes precedence over BindNetwork
+		if s.BindAddress != nil {
```

Comment:
was hesitating if it will be better to use `if s.BindAddress.To16() != nil ` , it will guarantee that is a valid IP address
https://golang.org/pkg/net/#IP.To16